Only in emacs-18.59/etc: DOC
Only in emacs-18.59/etc: DOC-18.59.1
Files emacs-18.59.orig/etc/ctags and emacs-18.59/etc/ctags differ
Only in emacs-18.59/etc: cvtmail
Files emacs-18.59.orig/etc/digest-doc and emacs-18.59/etc/digest-doc differ
Only in emacs-18.59/etc: emacsclient
Only in emacs-18.59/etc: env
diff -ur emacs-18.59.orig/etc/etags.c emacs-18.59/etc/etags.c
--- emacs-18.59.orig/etc/etags.c	2004-05-12 19:51:20.263954648 -0500
+++ emacs-18.59/etc/etags.c	2004-05-12 19:54:07.018604088 -0500
@@ -128,8 +128,6 @@
 
 char *savestr();
 char *savenstr ();
-char *rindex();
-char *index();
 char *concat ();
 void initbuffer ();
 long readline ();
@@ -1380,7 +1378,6 @@
      char *defenv;
 {
   register char *env, *p;
-  extern char *savenstr (), *index ();
 
   struct TEX_tabent *tab;
   int size, i;
@@ -1530,47 +1527,6 @@
   return dp;
 }
 
-/*
- * Return the ptr in sp at which the character c last
- * appears; NULL if not found
- *
- * Identical to v7 rindex, included for portability.
- */
-
-char *
-rindex(sp, c)
-     register char *sp, c;
-{
-  register char *r;
-
-  r = NULL;
-  do
-    {
-      if (*sp == c)
-	r = sp;
-    } while (*sp++);
-  return(r);
-}
-
-/*
- * Return the ptr in sp at which the character c first
- * appears; NULL if not found
- *
- * Identical to v7 index, included for portability.
- */
-
-char *
-index(sp, c)
-     register char *sp, c;
-{
-  do
-    {
-      if (*sp == c)
-	return (sp);
-    } while (*sp++);
-  return (NULL);
-}
-
 /* Print error message and exit.  */
 
 fatal (s1, s2)
Only in emacs-18.59/etc: fakemail
Files emacs-18.59.orig/etc/make-docfile and emacs-18.59/etc/make-docfile differ
Only in emacs-18.59/etc: movemail
Only in emacs-18.59/etc: server
Files emacs-18.59.orig/etc/sorted-doc and emacs-18.59/etc/sorted-doc differ
Files emacs-18.59.orig/etc/wakeup and emacs-18.59/etc/wakeup differ
Only in emacs-18.59/etc: yow
diff -ur emacs-18.59.orig/lisp/version.el emacs-18.59/lisp/version.el
--- emacs-18.59.orig/lisp/version.el	2004-05-12 19:51:09.078655072 -0500
+++ emacs-18.59/lisp/version.el	2004-05-12 19:57:25.812382840 -0500
@@ -20,7 +20,7 @@
 
 ;; The following line is modified automatically
 ;; by loading inc-version.el, each time a new Emacs is dumped.
-(defconst emacs-version "18.59.0" "\
+(defconst emacs-version "18.59.1" "\
 Version numbers of this version of Emacs.")
 
 (defconst emacs-build-time (current-time-string) "\
Only in emacs-18.59/src: abbrev.o
Only in emacs-18.59/src: alloc.o
Only in emacs-18.59/src: buffer.o
Only in emacs-18.59/src: bytecode.o
Only in emacs-18.59/src: callint.o
Only in emacs-18.59/src: callproc.o
Only in emacs-18.59/src: casefiddle.o
Only in emacs-18.59/src: cm.o
Only in emacs-18.59/src: cmds.o
Only in emacs-18.59/src: config.h
Only in emacs-18.59/src: data.o
Only in emacs-18.59/src: dired.o
Only in emacs-18.59/src: dispnew.o
Only in emacs-18.59/src: doc.o
Only in emacs-18.59/src: doprnt.o
Only in emacs-18.59/src: editfns.o
Only in emacs-18.59/src: emacs-18.59.1
Only in emacs-18.59/src: emacs.o
Only in emacs-18.59/src: eval.o
Only in emacs-18.59/src: fileio.o
Only in emacs-18.59/src: filelock.o
Only in emacs-18.59/src: filemode.o
Only in emacs-18.59/src: fns.o
Only in emacs-18.59/src: indent.o
Only in emacs-18.59/src: insdel.o
Only in emacs-18.59/src: keyboard.o
Only in emacs-18.59/src: keymap.o
Only in emacs-18.59/src: lastfile.o
diff -ur emacs-18.59.orig/src/lisp.h emacs-18.59/src/lisp.h
--- emacs-18.59.orig/src/lisp.h	2004-05-12 19:51:10.200484528 -0500
+++ emacs-18.59/src/lisp.h	2004-05-12 19:56:52.999371176 -0500
@@ -983,7 +983,7 @@
 
 extern void debugger ();
 
-extern char *malloc (), *realloc (), *getenv (), *ctime (), *getwd ();
+/* extern void *malloc (), *realloc (),*getenv (), *ctime (), *getwd (); */
 extern long *xmalloc (), *xrealloc ();
 
 #ifdef MAINTAIN_ENVIRONMENT
Only in emacs-18.59/src: lisp.h~
Only in emacs-18.59/src: lread.o
Only in emacs-18.59/src: macros.o
Only in emacs-18.59/src: malloc.o
Only in emacs-18.59/src: marker.o
Only in emacs-18.59/src: minibuf.o
Only in emacs-18.59/src: mocklisp.o
Only in emacs-18.59/src: paths.h
Only in emacs-18.59/src: pre-crt0.o
Only in emacs-18.59/src: print.o
Only in emacs-18.59/src: process.o
Only in emacs-18.59/src: regex.o
diff -ur emacs-18.59.orig/src/s-linux.h emacs-18.59/src/s-linux.h
--- emacs-18.59.orig/src/s-linux.h	2004-05-12 19:51:09.959521160 -0500
+++ emacs-18.59/src/s-linux.h	2004-05-12 20:03:12.614660880 -0500
@@ -253,7 +253,7 @@
 
 #define C_COMPILER gcc
 #define C_DEBUG_SWITCH -g
-#define C_OPTIMIZE_SWITCH -O2 -malign-loops=2 -malign-jumps=2 -malign-functions=2
+#define C_OPTIMIZE_SWITCH -O2 -falign-loops=2 -falign-jumps=2 -falign-functions=2
 #define OLDXMENU_OPTIONS CFLAGS=-O2 EXTRA=insque.o
 #define START_FILES pre-crt0.o /usr/lib/crt1.o /usr/lib/crti.o
 #define LIBS_DEBUG		/* override in config.h to include -lg */
Only in emacs-18.59/src: s-linux.h~
Only in emacs-18.59/src: scroll.o
Only in emacs-18.59/src: search.o
Only in emacs-18.59/src: syntax.o
Only in emacs-18.59/src: sysdep.o
Only in emacs-18.59/src: temacs
Only in emacs-18.59/src: term.o
Only in emacs-18.59/src: terminfo.o
Only in emacs-18.59/src: undo.o
Only in emacs-18.59/src: unexelf.o
Only in emacs-18.59/src: window.o
Only in emacs-18.59/src: x11fns.o
Only in emacs-18.59/src: x11term.o
Only in emacs-18.59/src: xdisp.o
Only in emacs-18.59/src: xemacs
Only in emacs-18.59/src: xmakefile
Only in emacs-18.59/src: xmakefile~
